sdk/identity/azure-identity/test/ut/token_credential_impl_test.cpp

```diff
@@ -689,6 +1048,26 @@ TEST(TokenCredentialImpl, MaxValues)
           "{\"token\": \"x\",\"exp_in\":2147483648}", "token", "exp_in", "exp_at")),
       std::exception);
 
+  // 'ref_in' negative
+  EXPECT_THROW(
+      static_cast<void>(TokenCredentialImpl::ParseToken(
+          "{\"token\": \"x\",\"ref_in\":-1}", "token", "exp_in", "exp_at", "ref_in")),
```

Comment:
In these new cases, here and below, can you also provide the value for `exp_in` like below, to demonstrate that it is actually a `ref_in` parsing logic that throws, and not just because `exp_in` was missing. Plus, it is probably not likely to encounter a token response with no `exp_in`, but with `ref_in`.
```suggestion
          "{\"token\": \"x\", \"exp_in\":42,\"ref_in\":-1}", "token", "exp_in", "exp_at", "ref_in")),
```